_data-prepper/pipelines/configuration/processors/otel-metrics-raw.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: default
+title: otel metrics raw
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# otel metrics raw
+
+The `otel metrics raw` processor serializes a collection of `ExportMetricsServiceRequest` sent from the [otel-metrics-source]({{site.url}}{{site.baseurl}}//data-prepper/pipelines/configuration/sources/otel-metrics-source/) into a collection of string records.
```

Comment:
Switching the processor name to match how it appears in code.

```suggestion
The `otel_metrics_raw_processor` processor serializes a collection of `ExportMetricsServiceRequest` sent from the [otel-metrics-source]({{site.url}}{{site.baseurl}}//data-prepper/pipelines/configuration/sources/otel-metrics-source/) into a collection of string records.
```

Reply:
We should actually change this to "OTel metrics raw processor" without the tic marks or underscores. This is something that will also need to be addressed across the docset.